src/app/containers/FrontPageSection/utilities/storyRowsSplitter.jsx

```diff
@@ -0,0 +1,36 @@
+import { splitEvery, splitAt } from 'ramda';
+
+// Split the top row stories out into their own list
+const splitFirstRow = (items, isFirstSection) => {
+  const shouldHaveOneItemInFirstRow = isFirstSection || items.length % 4 === 1;
+  const shouldHaveTwoItemsInFirstRow = items.length % 4 > 1;
+  if (shouldHaveOneItemInFirstRow) {
+    return splitAt(1, items);
```

Comment:
What's the general consensus on `ramda` vs raw JavaScript? I'd say `splitEvery` is useful in this case, but JavaScript is more readable for `splitAt`.

```js
return [
  items.slice(0, 1),
  items.slice(1),
];
```

Reply:
I've done this and also replaced `take` similarly to be more widely understandable and reduce our bundle size slightly since we're importing the ramda definition (which is actually exactly the code you've written).

https://github.com/bbc/simorgh/pull/5139/commits/1661b56391334586223f05c362e00174211a721f and https://github.com/bbc/simorgh/pull/5139/commits/ef5bb5ee06dc2eecf3d305b6cca939f51f1a99ed.